api/PositionOptions.json

```diff
@@ -0,0 +1,540 @@
+              }
+            ],
+            "edge_mobile": [
+              {
+                "version_added": null
+              },
+              {
+                "version_added": true,
+                "notes": "Secure context not required"
+              }
+            ],
+            "firefox": [
+              {
+                "version_added": "55"
+              },
+              {
+                "version_added": "3.5",
+                "version_removed": "55",
+                "notes": "Secure context not required"
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "55"
+              },
+              {
+                "version_added": "4",
+                "version_removed": "55",
+                "notes": "Secure context not required"
+              }
+            ],
+            "ie": {
+              "version_added": "9",
+              "notes": "Secure context not required"
+            },
+            "opera": [
+              {
+                "version_added": true
+              },
+              {
+                "version_added": "16",
+                "version_removed": true,
+                "notes": "Secure context not required"
+              },
+              {
+                "version_added": "10",
+                "version_removed": "15",
+                "notes": "Secure context not required"
+              }
+            ],
+            "opera_android": [
+              {
+                "version_added": true
+              },
+              {
+                "version_added": "16",
+                "version_removed": true,
+                "notes": "Secure context not required"
+              },
+              {
+                "version_added": "10",
+                "version_removed": "15",
+                "notes": "Secure context not required"
+              }
+            ],
+            "safari": [
+              {
+                "version_added": true
+              },
+              {
+                "version_added": "5",
+                "version_removed": true,
+                "notes": "Secure context not required"
+              }
+            ],
+            "safari_ios": [
+              {
+                "version_added": true
+              },
+              {
+                "version_added": true,
+                "version_removed": true,
+                "notes": "Secure context not required"
+              }
+            ]
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "timeout": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PositionOptions/timeout",
+          "support": {
+            "webview_android": [
+              {
+                "version_added": "47"
+              },
+              {
+                "version_added": "5",
```

Comment:
Like the previous property, for this and the remaining entries in this support block, I think we don't need any of the secure context notes, since it's covered by the root of the interface.